src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/ElapsedEventArgs.cs

```diff
@@ -5,9 +5,13 @@ namespace System.Timers
 {
     public class ElapsedEventArgs : EventArgs
     {
-        internal ElapsedEventArgs(long fileTime)
+        internal ElapsedEventArgs(DateTime localTime)
+        {
+            SignalTime = localTime;
+        }
+
+        internal ElapsedEventArgs(long fileTime) : this(DateTime.FromFileTime(fileTime))
```

Comment:
This ctor can be deleted as there is no usage left (`Timer.cs` was the only one using it).

Reply:
Well, I'm too cautious and want to keep it compatible... I've fixed it as you wish.